tests/ulyanov_et_al_2016/test_loss.py

```diff
@@ -1,32 +1,147 @@
+import itertools
+
+import pytest
+
 import pytorch_testing_utils as ptu
+import torch
 from torch.nn.functional import mse_loss
 
+from pystiche import gram_matrix, ops
 from pystiche.image import extract_batch_size
+from pystiche.loss import PerceptualLoss
 from pystiche_papers.ulyanov_et_al_2016 import loss
 
 
-def test_ulyanov_et_al_2016_content_loss(
-    subtests, multi_layer_encoder_with_layer, batch_image
+def test_UlyanovEtAl2016FeatureReconstructionOperator(
+    subtests, multi_layer_encoder_with_layer, target_image, input_image
 ):
+    input_image = torch.cat((input_image, input_image), 0)
     multi_layer_encoder, layer = multi_layer_encoder_with_layer
     encoder = multi_layer_encoder.extract_encoder(layer)
-    target_image = batch_image
-    input_image = batch_image.flip((0,))
     target_enc = encoder(target_image)
```

Comment:
I think we should do the same with `target_image` and another `batch_size` so we can be sure that it has no effect.

Reply:
What do you mean by different `batch_size`? Due to the `mse_loss` the `batch _size` of the `target_image` can only have the size 1 or the same size as the `input_image`.

Reply:
Ah sorry, I thought we were testing the style loss, where the `batch_size` is irrelevant :facepalm:. In that case, go for the same `batch_size` as the `input_image`.